From 7968866356e2d7bd01d97aa47f0e124357066094 Mon Sep 17 00:00:00 2001
From: Fabien Marotte <fabien.marotte@intel.com>
Date: Thu, 14 Jun 2012 14:43:18 +0200
Subject: [PORT FROM MAIN] Revert "wlan: set default wifi regulatory domain v2"

BZ: 41553

This reverts commit ca518758fa695af80e6aacd1bc82da4fcfa404b1.
This commit is generating a regression.

Change-Id: I14aeaedcc85cd9df69c2bd60bafe2b6e8b553703
Orig-Change-Id: I78d0ed4b2b17c2835880c9d1003ebfc31699b107
Signed-off-by: Fabien Marotte <fabien.marotte@intel.com>
Reviewed-on: http://android.intel.com:8080/53149
Reviewed-by: Regairaz, Julie <julie.regairaz@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Signed-off-by: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Reviewed-on: http://android.intel.com:8080/59014
Reviewed-on: http://android.intel.com:8080/77965
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wl12xx-compat/AndroidWl12xxCompat.mk |    4 +---
 wl12xx-compat/Makefile               |    3 +--
 wl12xx-compat/net/wireless/reg.c     |    4 ----
 3 files changed, 2 insertions(+), 9 deletions(-)

diff --git a/wl12xx-compat/AndroidWl12xxCompat.mk b/wl12xx-compat/AndroidWl12xxCompat.mk
index d39deda..5ad5c2f 100644
--- a/wl12xx-compat/AndroidWl12xxCompat.mk
+++ b/wl12xx-compat/AndroidWl12xxCompat.mk
@@ -1,8 +1,6 @@
 # This makefile is included from vendor/intel/*/AndroidBoard.mk.
 
-REGDOM=$(word 2,$(subst _, ,$(word 1,$(PRODUCT_LOCALES))))
-
 .PHONY: build_wl12xx-compat
 build_wl12xx-compat: build_kernel
-	TARGET_TOOLS_PREFIX="$(ANDROID_BUILD_TOP)/$(TARGET_TOOLS_PREFIX)" vendor/intel/support/wl12xx-compat-build.sh -c $(CUSTOM_BOARD) -r $(REGDOM)
+	TARGET_TOOLS_PREFIX="$(ANDROID_BUILD_TOP)/$(TARGET_TOOLS_PREFIX)" vendor/intel/support/wl12xx-compat-build.sh -c $(CUSTOM_BOARD)
 
diff --git a/wl12xx-compat/Makefile b/wl12xx-compat/Makefile
index 9a4af15..8863292 100644
--- a/wl12xx-compat/Makefile
+++ b/wl12xx-compat/Makefile
@@ -40,8 +40,7 @@ CFLAGS += \
         -DCOMPAT_BASE_TREE="\"$(shell cat compat_base_tree)\"" \
         -DCOMPAT_BASE_TREE_VERSION="\"$(shell cat compat_base_tree_version)\"" \
         -DCOMPAT_PROJECT="\"Compat-wireless\"" \
-        -DCOMPAT_VERSION="\"$(shell cat compat_version)\"" \
-        -DCOMPAT_REGDOM="\"$(shell cat compat_regdom)\""
+        -DCOMPAT_VERSION="\"$(shell cat compat_version)\""
 
 # These exported as they are used by the scripts
 # to check config and compat autoconf
diff --git a/wl12xx-compat/net/wireless/reg.c b/wl12xx-compat/net/wireless/reg.c
index 55745aa..c45c8b7 100644
--- a/wl12xx-compat/net/wireless/reg.c
+++ b/wl12xx-compat/net/wireless/reg.c
@@ -153,11 +153,7 @@ static const struct ieee80211_regdomain world_regdom = {
 static const struct ieee80211_regdomain *cfg80211_world_regdom =
 	&world_regdom;
 
-#ifdef COMPAT_REGDOM
-static char *ieee80211_regdom = COMPAT_REGDOM;
-#else
 static char *ieee80211_regdom = "00";
-#endif /* COMPAT_REGDOM */
 static char user_alpha2[2];
 
 module_param(ieee80211_regdom, charp, 0444);
-- 
1.7.4.1

